internal/querynode/impl.go

```diff
@@ -1224,7 +1224,7 @@ func (node *QueryNode) GetDataDistribution(ctx context.Context, req *querypb.Get
 }
 
 func (node *QueryNode) SyncDistribution(ctx context.Context, req *querypb.SyncDistributionRequest) (*commonpb.Status, error) {
-	log := log.Ctx(ctx).With(zap.Int64("collectionID", req.GetCollectionID()), zap.String("channel", req.GetChannel()))
+	log := log.Ctx(ctx).With(zap.Int64("collectionID", req.GetCollectionID()), zap.String("channel", req.GetChannel()), zap.String("actions", req.String()))
```

Comment:
print request may cause log too long

Reply:
OK, so how about printing the length of actions?

Reply:
the only critical contents are segmentID, nodeID, version and the actionType